imports/email-templates/components/helpers.js

```diff
@@ -1,6 +1,10 @@
 import URL from 'url'
 import { URL as url2 } from 'meteor/url'
 
+export function address (userId) {
```

Comment:
Not getting why it is named `address`. How about `resolveUserName`?
Also, `userId` should be renamed to something like `user`